### DISCUSSION TITLE
Getting all the running worker threads in nodejs

### DISCUSSION THREAD

**Discussion by mrityunjay-1** [nodejs/help#4972] [Q&A] [answered]
I created a worker thread that is running fine. But from another route i walt to get all the worker threads to stop them. How can i do this ?

**Comment by nagayev** [comment]
You can do something like this:
```js
const workers = [];
const w = new Worker(some_params);
workers.push(w);
```

**Comment by mrityunjay-1** [comment, marked as answer]
Thanks Marat Nagayev

**Reply by nagayev** [reply]
@mrityunjay-1 Please mark my reply as an answer (green button).
I'm trying to archive Galaxy Brain archiment)
